src/main/java/ru/ayaz/Video.java

```diff
@@ -0,0 +1,38 @@
+package ru.ayaz;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+class Video {
+
+    private String videoName;
```

Comment:
I wouldn't prefix these with "video" - there is no need for it. These are things that compose Video object. "name" by default would mean "name of the video", etc.